devicesimulation/single-vm/docker-compose.dotnet.yml

```diff
@@ -0,0 +1,57 @@
+version: '2'
+
+services:
+  reverseproxy:
+    image: azureiotpcs/remote-monitoring-nginx:1.0.0-preview.3
+    ports:
+      - "80:80"
+      - "443:443"
+    depends_on:
+      - webui
+      - auth
+      - devicesimulation
+    volumes:
+      - /app/certs:/app/certs:ro
+
+  webui:
+    image: azureiotpcs/pcs-remote-monitoring-webui:1.0.0-preview.3
+    depends_on:
+      - auth
+      - devicesimulation
+    volumes:
+      - /app/webui-config.js:/app/build/webui-config.js:ro
+
+  auth:
+    image: azureiotpcs/pcs-auth-dotnet:1.0.0-preview.3
+    environment:
+      - PCS_AUTH_ISSUER
+      - PCS_AUTH_AUDIENCE
+      - PCS_AUTH_REQUIRED
+      - PCS_CORS_WHITELIST
+      - PCS_APPLICATION_SECRET
+
+  devicesimulation:
+    image: azureiotpcs/device-simulation-dotnet:1.0.0-preview.3
```

Comment:
they created a separate branch 'azure-iot-pcs-simulation' in device-simulation-dotnet repo. we might need to use a different image built from that branch. if there is no changes  comparing to tag '1.0.0-preview.3', then we should keep it.

Reply:
As we declared in PR, it is just a placeholder, the image tag will be updated frequently in future developing. At least, use an existing rather a nonexist current tag will not break the deployment

Reply:
agree. remember to update it when new image is used.

Reply:
let's put the right value please. If tags are missing, it's a good reason to create them now